src/components/VAudioTrack/layouts/VBoxLayout.vue

```diff
@@ -17,7 +17,7 @@
               hide-name
               :license="audio.license"
             />
-            <div>{{ $t(`audio-categories.${audio.category}`) }}</div>
+            <div>{{ $t(`filters.audio-categories.${audio.category}`) }}</div>
```

Comment:
This is probably my naivete with Vue, but how was this working before without it being fully qualified?

Reply:
The old keys existed but I'm deleting those in this PR because they were repeated and we don't need that redundancy here 😄